README.md

```diff
@@ -331,6 +333,9 @@ See the corresponding directories for information on how to use the provided con
 
 # Changelog
```

Comment:
Looking at the history, the release notes on GitHub are pretty consistently good: https://github.com/openmm/openmmforcefields/releases

Might be worth adding a link to the header of this section?

Reply:
Sounds good, I do that for the next change log entry but it makes sense to highlight it here.

Reply:
Yeah, I figure the high-level link gets most of the value of adding per-release links like you did, but with less work